database/models.py

```diff
@@ -12,27 +16,31 @@ class Author(db.Document):
     updatedOn = db.DateTimeField(default=datetime.datetime.utcnow)
     meta = {"collection": "authors"}
 
-    def to_json(self):
+    def to_json(self, *args, **kwargs):
```

Comment:
Do we make use of *args, **kwargs here? I don't think so. If not, then remove these.